proxy/core/src/main/scala/io/cloudstate/proxy/AkkaActorRegisterFeature.scala

```diff
@@ -8,26 +8,25 @@ import scala.collection.JavaConverters._
 
 @AutomaticFeature
 final class AkkaActorRegisterFeature extends Feature {
-
+  val akkaActorClass = classOf[akka.actor.Actor]
   override def duringAnalysis(access: Feature.DuringAnalysisAccess) =
-    reachableSubtypes(access, access.findClassByName("akka.akka.Actor")).foreach { cls =>
-      for {
-        context <- getDeclaredField(cls, "context")
-        self <- getDeclaredField(cls, "self")
-      } {
-        RuntimeReflection.register(cls)
-        RuntimeReflection.register( /* finalIsWritable = */ true, context, self)
-        RuntimeReflection.register(cls.getDeclaredFields: _*)
-        RuntimeReflection.register(cls.getDeclaredConstructors: _*)
+    if (access.isReachable(akkaActorClass)) {
```

Comment:
@dwijnand Rewrote this to avoid having the runtime to create a lot of NPEs at analysis time

Reply:
SGTM!  I need to read what "isReachable" means.

Reply:
I figured that if the base type is not reachable, then there are no subtypes either, since reachability needs to be transitive.

Reply:
Also, it was the lack of rachability testing of the base types paired with nulls in the returned collection of `reachableSubtypes´ which caused the NPEs you were seeing (and catching).

Reply:
Awesome stuff.